project.yaml

```diff
@@ -742,6 +745,7 @@ Layer:
                 foot,
                 bicycle,
                 tracktype,
+                CASE WHEN surface IN ('unpaved', 'compacted', 'dirt', 'earth', 'fine_gravel', 'grass', 'grass_paver', 'gravel', 'ground', 'mud', 'pebblestone', 'salt', 'sand', 'woodchips', 'clay') THEN 'unpaved' ELSE 'paved' END AS surface,
```

Comment:
It would be better to use a name other than surface for this, as it's a normalized surface, and we may want to consider tracktype in the future.